Admin Guide Fix /  master-fingerprint.html (#3032)
uyuni-project/uyuni-docs: modules/administration/pages/master-fingerprint.adoc

[[security-master-fingerprint]]
= Set up a Client to Master Validation Fingerprint

In highly secure network configurations you may wish to ensure your {salt} clients are connecting a specific master.

> To set up validatfrom client to master start by entering the master's fingerprint within the configuration file:

Reviewer: ```suggestion
To set up validation from client to master start by entering the master's fingerprint within the configuration file:
```

> In highly secure network configurations you may wish to ensure your {salt} clients are connecting a specific master.
> To set up validation from client to master start by entering the master's fingerprint within the configuration file:
> 
> * [path]``/etc/salt/minion.d/custom.conf`` in cases of using classic {salt} minion in your client, or
> * [path]``/etc/venv-salt-minion/minion.d/custom.conf`` in case of using {salt} Bundle in your client

Reviewer 2: These mentioned `custom.conf` files are not existing by default on a system, the user should have to create.

So I would rephrase to:

```suggestion
In highly secure network configurations you may wish to ensure your {salt} clients are connecting a specific master.
To set up validation from client to master start by entering the master's fingerprint within a {salt} minion configuration file:

* [path]``/etc/salt/minion.d/custom.conf`` in cases of using classic {salt} minion in your client, or
* [path]``/etc/venv-salt-minion/minion.d/custom.conf`` in case of using {salt} Bundle in your client
```

----
+

> On your client, open the [path]``/etc/salt/minion`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.

Reviewer 2: For consistency, we probably should mention `/etc/salt/minion.d/custom.conf` instead of `/etc/salt/minion`:

```suggestion
On your client, open the [path]``/etc/salt/minion.d/custom.conf`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.
```

Uncomment the following line and enter the master's fingerprint replacing the example fingerprint:
+

----
master_finger: 'ba:30:65:2a:d6:9e:20:4f:d8:b2:f3:a7:d4:65:11:13'
----

. Restart the salt-minion or venv-salt-minion service.
  For venv-salt-minion, run:
+

----

> systemctl restart vent-salt-minion

Reviewer 2: Typo:

```suggestion
systemctl restart venv-salt-minion
```

----

> In case of using {salt} Bundle, instead of [literal]``salt-minion`` use [literal]``venv-salt-minion``.

Reviewer 2: This statement should be the opposite, isn't it? As we already provided the example for `venv-salt-minion`:

Or maybe we should keep this statement here, but then provide the above example of restarting the service with the classic Salt example: `systemctl restart salt-minion`

----
+

On your client, open the [path]``/etc/salt/minion.d/custom.conf`` or [path]``/etc/venv-salt-minion/minion.d/custom.conf`` configuration file.

> Uncomment the following line and enter the master's fingerprint replacing the example fingerprint:

Reviewer 2: This we should also change, as the file is created by the user then it is empty, so the user cannot "uncomment the following line". We should ask them to add the file
